docker-compose.yml

```diff
@@ -4,10 +4,35 @@ services:
   web:
     build: .
     command: "yarn start:dev"
+    environment:
+      DATABASE_HOST: db
+      DATABASE_PORT: 3306
+      DATABASE_USER: user
+      DATABASE_PASSWORD: password
+      DATABASE_NAME: ppaa
+      DATABASE_LOGGING: true
     ports:
       - 3000:3000
     volumes:
       - .:/usr/src/app
       - node_modules:/usr/src/app/node_modules
+    depends_on:
+      - db
+    links:
+      - db:db
+  db:
+    image: mysql:8.3
+    restart: always
```

Comment:
Can you change the restart always for an unless-stopped? (sorry it just annoys me that the container is running when I turn on the machine)